docs/source/index.rst

```diff
@@ -42,7 +42,6 @@ DAML SDK documentation
    :caption: Building applications
 
    app-dev/app-arch
-   daml2js/index
```

Comment:
I guess this is intentional to only have a single entry point? Sounds reasonable to me.

Reply:
Yes, that's intended for exactly that reason. 😃